src/PaginationResult.php

```diff
@@ -61,19 +61,10 @@ public function getIterator()
      * Count records.
      *
      * @return int
+     * @see https://wiki.php.net/rfc/counting_non_countables
      */
     public function count()
     {
-        // @codeCoverageIgnoreStart
-        if (
-            !$this->records instanceof \Countable
-            && !is_array($this->records)
-            && version_compare(PHP_VERSION, '7.2.0', '>=')
-        ) {
-            // PHP: rfc:counting_non_countables https://wiki.php.net/rfc/counting_non_countables
-            trigger_error('count(): Parameter must be an array or an object that implements Countable', E_USER_WARNING);
-        }
-        // @codeCoverageIgnoreEnd
         return count($this->records);
```

Comment:
Just calling `count($this->records)` will handle argument as expected.

Reply:
LGTM👍